### ISSUE TITLE
Reverting a Commit in Episode 5 uses "group repository"

### ISSUE THREAD

**Issue by alex-konovalov** [swcarpentry/git-novice#370]
"Reverting a Commit" challenge in Episode 5 precedes remotes which will be introduced in Episode 7. Therefore the text 
```
> Jennifer is collaborating on her Python script with her colleagues and
> realises her last commit to the group repository is wrong and wants to
> undo it.  Jennifer needs to undo correctly so everyone in the group
> repository gets the correct change. 
```
may be confusing since it's not clear yet what does the "group repository" mean. Also, the list of commands in this challenge does not include pushing to the remote repository (maybe it's assumed that the group repository in this challenge is on a shared filesystem?).

Perhaps this challenge could be moved to some later episode when `push` and `pull` commands are already introduced. Alternatively, it could stay here, but the use case could be changed from the group one to individual - how to revert some old commit is useful knowledge in this case anyway.

**Comment by tylerkelly13** [comment]
Hi Alex,
Not sure about moving it, but I have updated the challenge to (hopefully) make it clearer.

**Comment by iglpdc** [comment]
Closed by #434.
